uint16[] channels
uint8 sbus_state